include/boost_dynamic_bitset_fwd.h

```diff
@@ -6,6 +6,7 @@
 namespace boost
 {
 #ifndef BOOST_DYNAMIC_BITSET_FWD_HPP
+#define BOOST_DYNAMIC_BITSET_FWD_HPP
 template <typename Block = unsigned long, typename Allocator = std::allocator<Block>> class dynamic_bitset;
```

Comment:
what is the addition for?

Reply:
this header file is in the boost package. but somehow, someone added it to here. Then it could cause conflict with the header file in the boost package